src/scss/custom/_stepper.scss

```diff
@@ -0,0 +1,36 @@
+.stepper__step {
+  color: $primary;
+  font-weight: 500;
+  &__value {
+    color: $primary;
+    border-color: $primary !important;
+    height: $spacer * 3;
+    width: $spacer * 3;
+    position: relative;
+  }
+
+  &--active {
+    .stepper__step__value {
+      color: $white;
+      background-color: $primary;
+      border: 2px;
+    }
+  }
+
+  &--todo {
+    color: $gray-400;
+    .stepper__step__value {
+      border-color: $gray-400 !important;
+      background-color: $gray-400;
+      color: $white;
+    }
+  }
+}
+
+.line {
+  border-top: 2px solid #{$gray-400};
+  -ms-flex: auto;
+  flex: auto;
+  height: 0;
+  min-width: 32px;
```

Comment:
I wonder if only the last couple of `px` values can come from existing vars? @josephrace @BRHGreen